crates/storage/src/lazy/mapping.rs

```diff
@@ -130,16 +130,6 @@ where
     V: Packed,
     KeyType: StorageKey,
 {
-    /// Insert the given `value` to the contract storage.
-    #[inline]
-    pub fn insert<Q, R>(&mut self, key: Q, value: &R)
-    where
-        Q: scale::EncodeLike<K>,
-        R: Storable + scale::EncodeLike<V>,
-    {
-        ink_env::set_contract_storage(&(&KeyType::KEY, key), value);
-    }
-
```

Comment:
Shouldn't we now rename `insert_return_size` to `insert` as per https://github.com/paritytech/ink/pull/1224#issuecomment-1107803882